org-brain.el

```diff
@@ -1999,7 +2041,7 @@ Helper function for `org-brain-visualize'."
     (insert "SELECTED:")
     (dolist (selection (sort (copy-sequence org-brain-selected) org-brain-visualize-sort-function))
       (insert "  ")
-      (org-brain-insert-visualize-button selection 'org-brain-selected))
+      (org-brain-insert-visualize-button selection 'org-brain-pinned))
```

Comment:
Hrm, this is a hold-over from the first implementation of the selected list, which just used the same face as the pinned entries, because selected items were also at the top of the screen along with pinned items. But I agree org-brain-button is more reasonable.